diff --git a/src/strands/tools/executors/_executor.py b/src/strands/tools/executors/_executor.py
index 87c38990d..1fa0c6ebc 100644
--- a/src/strands/tools/executors/_executor.py
+++ b/src/strands/tools/executors/_executor.py
@@ -77,6 +77,7 @@ class ToolExecutor(abc.ABC):
             {
                 "model": agent.model,
                 "messages": agent.messages,
+                "agent": agent,
                 "system_prompt": agent.system_prompt,
                 "tool_config": ToolConfig(  # for backwards compatibility
                     tools=[{"toolSpec": tool_spec} for tool_spec in agent.tool_registry.get_all_tool_specs()],
