diff --git a/src/strands/agent/agent.py b/src/strands/agent/agent.py
index 677ecb876..956c246ce 100644
--- a/src/strands/agent/agent.py
+++ b/src/strands/agent/agent.py
@@ -457,7 +457,7 @@ class Agent:
                 content: list[ContentBlock] = [{"text": prompt}] if isinstance(prompt, str) else prompt
                 self._append_message({"role": "user", "content": content})
 
-            events = self.model.structured_output(output_model, self.messages)
+            events = self.model.structured_output(output_model, self.messages, system_prompt=self.system_prompt)
             async for event in events:
                 if "callback" in event:
                     self.callback_handler(**cast(dict, event["callback"]))
