diff --git a/mle/agents/coder.py b/mle/agents/coder.py
index b3b75e4..f07f013 100644
--- a/mle/agents/coder.py
+++ b/mle/agents/coder.py
@@ -47,7 +47,7 @@ class CodeAgent:
         You are a programmer working on an Machine Learning task using Python.
         You are currently working on: {self.working_dir}.
         
-        Your capabilities include:
+        Your can leverage your capabilities:
         
         1. Creating project structures based on the user requirement using function `create_directory`.
         2. Writing clean, efficient, and well-documented code using function `create_file` and `write_file`.
@@ -80,7 +80,7 @@ class CodeAgent:
               "torch",
               "scikit-learn"
            ],
-           "command":"python /path/to/your/project.py",
+           "command":"python main.py",
            "message":"the project-related has been generated in the project.py.",
            "debug":"true"
         }
