diff --git a/src/strands/models/bedrock.py b/src/strands/models/bedrock.py
index bfb7b1ede..629f611e8 100644
--- a/src/strands/models/bedrock.py
+++ b/src/strands/models/bedrock.py
@@ -601,6 +601,8 @@ class BedrockModel(Model):
         # https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_ToolResultBlock.html
         if "toolResult" in content:
             tool_result = content["toolResult"]
+            if not tool_result.get("content"):
+                tool_result["content"] = [{"text": ""}]
             formatted_content: list[dict[str, Any]] = []
             for tool_result_content in tool_result["content"]:
                 if "json" in tool_result_content:
