diff --git a/src/strands/__init__.py b/src/strands/__init__.py
index 6026d4240..97cd43c28 100644
--- a/src/strands/__init__.py
+++ b/src/strands/__init__.py
@@ -3,11 +3,14 @@
 from . import agent, models, telemetry, types
 from .agent.agent import Agent
 from .agent.base import AgentBase
+from strands.plugins.plugin import Plugin
+from strands.plugins.plugin import Plugin
 from .event_loop._retry import ModelRetryStrategy
 from .tools.decorator import tool
 from .types.tools import ToolContext
 
 __all__ = [
+    "Plugin",
     "Agent",
     "AgentBase",
     "agent",
diff --git a/src/strands/hooks/__init__.py b/src/strands/hooks/__init__.py
index 96c7f577b..f87ef7a64 100644
--- a/src/strands/hooks/__init__.py
+++ b/src/strands/hooks/__init__.py
@@ -46,6 +46,7 @@ from .events import (
     MultiAgentInitializedEvent,
 )
 from .registry import BaseHookEvent, HookCallback, HookEvent, HookProvider, HookRegistry
+from strands.plugins.plugin import Plugin
 
 __all__ = [
     "AgentInitializedEvent",
@@ -58,6 +59,7 @@ __all__ = [
     "MessageAddedEvent",
     "HookEvent",
     "HookProvider",
+"Plugin"
     "HookCallback",
     "HookRegistry",
     "HookEvent",
